### PR TITLE
Switch Sullen-Ural connector to v2 format

### PR DESCRIPTION
**Pull request by alexesprit**

### CODE REVIEW

File: connectors/v2/sullen-ural.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+		var artist = null;
+		var track = null;
+
+		if (separator !== null) {
+			artist = text.substr(0, separator.index);
+			track = text.substr(separator.index + separator.length);
+		}
+
+		return {artist: artist, track: track};
```
Comment (inverse) [inline connectors/v2/sullen-ural.js]: I would change this to have a single point of return - Perhaps init the track/artist vars outside of the first If. What do you think?

File: connectors/v2/sullen-ural.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = $('#tracks-async').length ? '#tracks-async' : ($('.band-col-right').length ? '.band-col-right' : '.album-tracks');
```
Comment (inverse) [inline connectors/v2/sullen-ural.js]: Could we make this a bit more readable? Having multiple ternary operators in a single statement is pretty hard to read

File: connectors/v2/sullen-ural.js (hunk 1 above)
Comment (alexesprit) [inline connectors/v2/sullen-ural.js]: Sorry, didn't understand what you mean.

File: connectors/v2/sullen-ural.js (hunk 2 above)
Comment (alexesprit) [inline connectors/v2/sullen-ural.js]: You're totally right. I'll update the code soon.

**Comment by alexesprit** [comment]
Simplified setup of player selector.

`#tracks-async` and `.band-col-right` selectors are parents of `.album-tracks` one, so it makes no sense to use them.

**Comment by inverse** [comment]
Much nicer!

**Pull request merged**